task/01-strings-tasks.js

```diff
@@ -238,6 +276,11 @@ function encodeToRot13(str) {
  *   isString(new String('test')) => true
  */
 function isString(value) {
+    if(typeof value === 'undefined'||  value === null)
+        return false;
+    else if('string'===typeof value && value || 'object'===typeof value && Object.keys(value).length)
+    return true
+    else return false;
     throw new Error('Not implemented');
```

Comment:
a bad code style and, lot of extra code
all that you need: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof